### PR TITLE
Create B827EBFFFE553592.json

### PR DESCRIPTION
**Pull request by maraton84**

### CODE REVIEW

**Comment by gonzalocasas** [comment]
This isn't merged yet, don't you want to re-open?

**Comment by maraton84** [comment]
Hello,
yes, I want.
Regards, Tamas

    On Thursday, May 18, 2023 at 01:16:35 PM GMT+2, Gonzalo Casas ***@***.***> wrote:  
 
 


This isn't merged yet, don't you want to re-open?

File: B827EBFFFE553592.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE553592",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```
Comment (gonzalocasas) [inline B827EBFFFE553592.json:6]: This is the old V2 router, you need to update to eu1.cloud.thethings.network

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
Please update as requested below and then we'll merge

File: B827EBFFFE553592.json (hunk 1 above)
Comment (maraton84) [reply B827EBFFFE553592.json]: I've changed it. Cold you please check it? Do I need to make something else? Thanks in advance!

Comment (gonzalocasas) [reply B827EBFFFE553592.json]: Done and merged!

**Pull request merged**